composer.json

```diff
@@ -21,13 +21,13 @@
     "require": {
         "php": "^7.1",
         "sonata-project/cache": "^2.0.0",
-        "symfony/config": "^3.4 || ^4.0",
-        "symfony/dependency-injection": "^3.4 || ^4.0",
-        "symfony/filesystem": "^3.4 || ^4.0",
-        "symfony/http-foundation": "^3.4.35 || ^4.0",
-        "symfony/http-kernel": "^3.4 || ^4.0",
+        "symfony/config": "^3.4 || ^4.0 || ^5.0",
+        "symfony/dependency-injection": "^3.4 || ^4.0 || ^5.0",
+        "symfony/filesystem": "^3.4 || ^4.0 || ^5.0",
+        "symfony/http-foundation": "^3.4.35 || ^4.0 || ^5.0",
+        "symfony/http-kernel": "^3.4 || ^4.0 || ^5.0",
         "symfony/process": "^3.4 || ^4.0",
```

Comment:
The process component can't be upgraded to 5 without a BC break (due to the component not accepting string arguments anymore), so this should be upgraded separately for master.